app/bundles/LeadBundle/Model/ListModel.php

```diff
@@ -602,6 +602,8 @@ public function getChoiceFields()
                     'list' => [
                         0 => $this->translator->trans('mautic.core.form.no'),
```

Comment:
This is confusing to me. Yes/No/Subscribed/Unsubscribed. What does yes/no refer to? Maybe it's just a labeling issue.

Reply:
Maybe you can help me with labeling
- in past we save mautic notification OneSignal IDs related to user to push_ids table
- times ago @dongilbert  added new column active to this table
- Yes option means all contacts with at least one notification OneSignal ID
- No option means all contacts without at least one notification OneSignal ID
- Subscribed - means all contacts with at least one notification OneSignal IDs, but with flag active  1 (after subscribe, after send notification we know if recepients OneSignal ID is active)
- Unsubscribe - we have notifciation OneSignal ID for contact, but meanwhile contact disable notification

Reply:
The reason why we wanna filter Subscribed/Unsubscribed  that now we can send 3000 notifications, but real number receive notifications is less. We need real number of active devices (OneSignal Contacts)